ctru-rs/src/services/apt.rs

```diff
@@ -90,3 +132,48 @@ impl Drop for Apt {
         unsafe { ctru_sys::aptExit() };
     }
 }
+
+/// Can launch other applications when the current one exits.
+pub struct Chainloader<'a> {
+    _apt: &'a Apt,
+}
+
+impl<'a> Chainloader<'a> {
+    /// Gets a handle to the chainloader
+    pub fn new(apt: &'a Apt) -> Self {
+        Self { _apt: apt }
+    }
+
+    /// Checks if the chainloader is set
+    #[doc(alias = "aptIsChainload")]
+    pub fn is_set(&self) -> bool {
+        // static funtion not exported
+        unsafe { (ctru_sys::envGetSystemRunFlags() & ctru_sys::RUNFLAG_APTCHAINLOAD) != 0 }
+    }
+
+    /// Clears the chainloader state.
+    #[doc(alias = "aptClearChainloader")]
+    pub fn clear(&mut self) {
+        unsafe { ctru_sys::aptClearChainloader() }
+    }
+
+    /// Configures the chainloader to launch a specific application.
+    ///
+    /// See also [super::am::Title]
```

Comment:
Nit: we usually use code blocks in doc links.
```suggestion
    /// See also [`Title`](crate::services::am::Title]
```